scripts/boxstartwin7.bat

```diff
@@ -0,0 +1,10 @@
+@ECHO OFF
```

Comment:
I'd rather not have PSv4 installed. I'd like to keep these base boxes as "bare" as possible. This would prevent one from reproing many issues you see on a "typical" win7 box. Hope that makes sense.